app/policies/follow_request_policy.rb

```diff
@@ -0,0 +1,32 @@
+class FollowRequestPolicy < ApplicationPolicy
+  attr_reader :user, :follow_request
+
+  def initialize(user, follow_request)
+    @user = user
+    @follow_request = follow_request
+  end
+
+  def index?
+    true
+  end
+
+  def show?
+    true
+  end
+
+  def create?
+    true
+  end
+
+  def update?
+    true
+  end
+
+  def destroy?
+    true
+  end
+
+  def set_follow_request
+    true
+  end
+end
```

Comment:
Think about who can do all this. Right now anyone can do these actions according to this which is not what we want.